test/sandbox/web3/getNodeInfo.ts

```diff
@@ -1,5 +1,5 @@
-import { web3 } from 'helpers/web3'
 import Logger from 'helpers/Logger'
+import { web3 } from '../../helpers/web3'
```

Comment:
web3 is no longer exported from there

Reply:
That a new web3 helper used only for tests:

![image](https://user-images.githubusercontent.com/2352112/71515158-378caa80-28a2-11ea-9bdd-afbcf5607925.png)